data/tmux.conf

```diff
@@ -5,20 +5,16 @@ bind -n F1 list-keys
 
 set-option -s exit-unattached off
 set-option -g base-index 1
-set-option -g set-remain-on-exit on
```

Comment:
Something like this?
`if-shell` runs first statement if the condition is true, otherwise the second

```
if-shell '[ $(echo "$(tmux -V | cut -d" " -f2) >= 2.4" | bc) -eq 1 ]' \
    'set-option -g remain-on-exit on' \
    'set-option -g set-remain-on-exit on'
```